lib/App/Dochazka/REST/ResourceDefs.pm

```diff
@@ -492,6 +492,7 @@ Deletes the activity object whose AID is specified by the ':aid' URI parameter.
 This will work only if nothing in the database refers to this activity.
 
 =back
+
```

Comment:
The usefulness of running podchecker on `ResourceDefs.pm` is debatable, since this file contains incomplete snippets of POD that are concatenated together to make this: https://st.aticpan.org/source/SMITHFARM/App-Dochazka-REST-0.554/lib/App/Dochazka/REST/Docs/Resources.pm